spec/services/invoices/calculate_fees_service_spec.rb

```diff
@@ -616,6 +616,46 @@
       end
     end
 
+    context 'when subscription is paid in advance' do
+      let(:plan) do
+        create(:plan, interval: :monthly, pay_in_advance: true, amount_cents: 1000)
+      end
+
+      # let(:next_subscription) do
+      #   create(
+      #     :subscription,
+      #     plan: next_plan,
+      #     subscription_at: started_at.to_date,
+      #     started_at: terminated_at,
+      #     status: :active,
+      #     billing_time: :calendar,
+      #     previous_subscription: subscription,
+      #     customer: subscription.customer,
+      #   )
+      # end
+
+      let(:started_at) { DateTime.parse('07 Mar 2022') }
+      let(:timestamp) { started_at }
+
+      let(:subscription) do
+        create(
+          :subscription,
+          plan:,
+          subscription_at: started_at.to_date,
+          started_at:,
+          billing_time: :calendar,
+        )
+      end
+
+      let(:charge) do
+        create(:standard_charge, plan:, properties: { amount: 100 })
+      end
+
+      it 'updates the invoice with only the charge fees' do
+        result = invoice_service.call
```

Comment:
Lint/UselessAssignment: Useless assignment to variable - `result`. (https://rubystyle.guide#underscore-unused-vars)